### DISCUSSION TITLE
I would like to speak to you

### DISCUSSION THREAD

**Discussion by The-One123** [prometheus-lua/Prometheus#22] [Ideas]
Hey I’d like to speak to you could I please get your discord

**Comment by levno-710** [comment]
I created a Discord-Server for Prometheus: https://discord.gg/JqZPJf77NB

**Comment by sarahsturgeon** [comment]
The invite's expired; could we get a new one?

**Reply by levno-710** [reply]
https://discord.gg/U8h4d4Rf64
